src/bokeh/protocol/message.py

```diff
@@ -29,11 +29,11 @@
 
     header = {
         # these are required
-        'msgid'       : <str> # a unique id for the message
-        'msgtype'     : <str> # a message type, e.g. 'ACK', 'PATCH-DOC', etc
+        'msgid'       : <str>, # a unique id for the message
+        'msgtype'     : <str>, # a message type, e.g. 'ACK', 'PATCH-DOC', etc
 
         # these are optional
-        'num_buffers' : <int> # the number of additional buffers, if any
+        'num_buffers' : <int>, # the number of additional buffers, if any
```

Comment:
I don't think the commas increase clarity here, this is illustrative and not going to be copy-pasted so in that sense they just add clutter